src/public/dashboard/dashboard.php

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+ 
+	<title></title>
```

Comment:
You don't need to put html, head, title, body tags in all files, these files get included in the main index where they are already surrounded by those tags.